.travis.yml

```diff
@@ -128,7 +128,7 @@ install:
       fi;
 
 script:
-    - nosetests -a '!ultra_slow_test' --with-timer --timer-top-n 30 --verbosity=2 $COVERAGE
+    - nosetests -a '!ultra_slow_test' --ignore-files=^_\\w*_gui.py --ignore-files=_file_traits.py --ignore-files=_viewer.py --with-timer --timer-top-n 30 --verbosity=2 $COVERAGE
```

Comment:
can you see if these ignores can be added to our `setup.cfg` instead of going in here? That way things like `nosetests` and `make test` will pick them up, too.